public/programmerjs/receptiontransmission.js

```diff
@@ -26,6 +25,7 @@ var Paysage = window.Paysage || {};
     function emitData (data) {
       console.log(data);
       io.emit('code update', data);
+      window.location.hash = data.name;
```

Comment:
This really feels out of place. Regardless of the desired behavior, I don’t see why this module named « receptiontransmission » suddenly needs to change the current location…